### DISCUSSION TITLE
My application has not been approved.

### DISCUSSION THREAD

**Discussion by SmallCoral** [community/community#139805] [GitHub Education]
### Hi everyone,

My application has not been approved. I am a Chinese student. Due to the special network environment in China, I have to use VPN to apply all the time.
![Screenshot_20240926_135159_Chrome](https://github.com/user-attachments/assets/185a16b3-f34e-4051-83ae-3fab44297785)

**Comment by ferid333** [comment]
HI @SmallCoral 

It sounds like the issue could be related to using a VPN while applying for GitHub's Student Developer Pack. Sometimes, using a VPN may trigger security checks or block certain applications, especially if GitHub detects unusual network activity.

**Reply by SmallCoral** [reply]
I can't use GitHub without a VPN, is there any other solution?

**Reply by ferid333** [reply]
If a VPN isn't working reliably, using a proxy server might be a more stable alternative for accessing GitHub.

**Reply by SteveSWang47** [reply]
> If a VPN isn't working reliably, using a proxy server might be a more stable alternative for accessing GitHub.

What do you mean? We use VPN to access GitHub because of [the GFW](https://en.wikipedia.org/wiki/Great_Firewall), and GitHub says `You may not use a VPN. `

The VPN works reliably, but we get rejected because of it. Proxy servers won't help at all.
